plugin.xml

```diff
@@ -1,7 +1,7 @@
 <plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
 xmlns:android="http://schemas.android.com/apk/res/android"
            id="cordova-plugin-firebase-crash"
-      version="4.1.2">
+      version="4.2.0">
```

Comment:
Please rollback version change from PR. I'll publish new release with tag as soon as the PR will me merged.
```suggestion
      version="4.1.2">
```

Reply:
Done